mypy/checker.py

```diff
@@ -2711,9 +2711,14 @@ def get_isinstance_type(expr: Expression, type_map: Dict[Expression, Type]) -> T
 
             types.append(type)
 
```

Comment:
I think there are too many blank lines here. IMO the only blank line should be outside the for-loop, before the assert on line 2720 below.